### PR TITLE
Adding language selection to script.

### LINKED ISSUE #24: Spanish language pack
Create a Spanish language pack for the script.

### PR DESCRIPTION
**Pull request by vetom**
This fixes #24

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ exports.getAllCards = function () {
 
 exports.getAllTemplates = cards.getAllTemplates;
 
-exports.getScriptFromDeck = function (deck) {
-    return moderator.getScriptFromDeck(deck);
+exports.getScriptFromDeck = function (deck, language) {
```
Comment (wovalle) [inline index.js]: You could do `exports.getScriptFromDeck = moderator.getScriptFromDeck` directly

File: moderator.js [hunk 2]
```diff
@@ -2,17 +2,19 @@ var lang = require('./languages');
 var bindings = require('./bindings');
 var secuence = require('./secuence');
 
-exports.getScriptFromDeck = function (deck) {
+exports.getScriptFromDeck = function (deck, selectedLanguage) {
```
Comment (wovalle) [inline moderator.js:5]: Wouldn't be a better idea to add lang.english as default for selected language? In that case you don't need two variables (language && selected language) to do the same. 

```exports.getScriptFromDeck = function (deck, selectedLanguage  = lang.english)```

**Review by wovalle — CHANGES_REQUESTED** [review]

**Review by wovalle — APPROVED** [review]

File: moderator.js (hunk 2 above)
Comment (vetom) [reply moderator.js:5]: The idea with selectedLanguage it's for specifying the language, not the entire object.

**Comment by vetom** [comment]
This PR fixes #25 too!

**Pull request merged**